lib/ClassMover/Tests/Adapter/WorseTolerant/WorseTolerantMemberFinderTest.php

```diff
@@ -19,7 +19,8 @@ public function testFindMember(string $source, ClassMemberQuery $classMember, in
         $this->assertCount($expectedRiskyCount, $members->withoutClasses());
     }
 
-    public function provideFindMember()
+    /** @return array<array{0: string, 1: ClassMemberQuery, 2: int, 3?:int}> */
+    public function provideFindMember(): array
```

Comment:
note you can use the Phpactor code action "generate docblock return types" here to get 80% of the way, I would prefer multi line PHPDocs but won't stop a PR onit.

Reply:
Fair enough I'll keep that in mind. I like single line ones since they are less visually distracting. But shouldn't be a problem to switch. Or we add a code formatter rule for it, since the project uses both.